docs/docs/reference/crd-reference/app.md

```diff
@@ -165,14 +133,36 @@ spec:
 
 ## Differences between versions
 
-- The `spec.revision` field is introduced in v1alpha2.
-- The pre/post-deployment tasks and evaluations are not
-supported anymore in v1beta1.
+The synopsis of the `KeptnApp` resource
+is changed in `v1beta1` library:
+
+- The `apiVersion` field now references the `v1beta1` version
+  of the `lifecycle.keptn.sh` API.
+- `KeptnApp` must be
+  [autogenerated](../../guides/auto-app-discovery.md);
+  you cannot create it manually by applying a manifest.
+- The `name` and `namespace` fields are populated
```

Comment:
the name of the KeptnApp is taken from the annotations of the deployed k8s workloads. Namespace is the same as the k8s workloads as well. I would not mention KeptnAppContext here at all

Reply:
fixed